### DISCUSSION TITLE
lost sensors previous_accumulative_consumption_off_peak and previous_accumulative_consumption_peak

### DISCUSSION THREAD

**Discussion by lamixer** [BottlecapDave/HomeAssistant-OctopusEnergy#519] [Q&A] [answered]
November 3rd was the last day I had information for those sensors. Is this a known issue or just something weird for me? It is just reporting 0 now which is not correct. The entity still exists. Thanks.

**Comment by BottlecapDave** [comment, marked as answer]
It's probably related to https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/512 if you've updated recently

**Reply by lamixer** [reply]
Ah, sorry I missed seeing that one. Yes, probably the same issue. I'm on Intelligent Octopus Go. Thanks for the great work on this integration!
